File: namsor/typescript-node/src/GenderFullGeoExample.ts
import * as fs from 'fs';
import api from "namsor_client"
import models from "namsor_client"

const apiCaller = new api.PersonalApi();
apiCaller.setApiKey(api.PersonalApiApiKeys.api_key, "YOUR_API_KEY");

apiCaller.genderFullGeo(
  "Keith Haring", // fullName
  "US", // countryIso2
).then(response => {
  console.log(response.body);
}).catch(error => {
  console.log("Exception when calling PersonalApi#genderFullGeo:");
  console.log(error.body);
});

File: namsor/kotlin/src/main/kotlin/oseg/namsor_examples/GenderFullGeoExample.kt
package oseg.namsor_examples

import app.namsor.client.infrastructure.*
import app.namsor.client.apis.*
import app.namsor.client.models.*

import java.io.File
import java.time.LocalDate
import java.time.OffsetDateTime
import kotlin.collections.ArrayList
import kotlin.collections.List
import kotlin.collections.Map
import com.squareup.moshi.adapter

@ExperimentalStdlibApi
class GenderFullGeoExample
{
    fun genderFullGeo()
    {
        ApiClient.apiKey["X-API-KEY"] = "YOUR_API_KEY"

        try
        {
            val response = PersonalApi().genderFullGeo(
                fullName = "Keith Haring",
                countryIso2 = "US",
            )

            println(response)
        } catch (e: ClientException) {
            println("4xx response calling PersonalApi#genderFullGeo")
            e.printStackTrace()
        } catch (e: ServerException) {
            println("5xx response calling PersonalApi#genderFullGeo")
            e.printStackTrace()
        }
    }
}
